travis.sh

```diff
@@ -170,11 +170,12 @@ travis_time_end  # rosdep_install
 # Start prerelease, and once it finishs then finish this script too.
 # This block needs to be here (i.e. After rosdep is done) because catkin_test_results isn't available until up to this point.
 travis_time_start prerelease_from_travis_sh
-if [ "$PRERELEASE" == true ] && [ -e ${CI_SOURCE_PATH}/$CI_PARENT_DIR/ros_pre-release.sh ]; then 
-  ${CI_SOURCE_PATH}/$CI_PARENT_DIR/ros_pre-release.sh
-  catkin_test_results build && (echo 'ROS Prerelease Test went successful.'; exit 0) || error
+if [ "$PRERELEASE" == true ] && [ -e ${CI_SOURCE_PATH}/$CI_PARENT_DIR/ros_pre-release.sh ]; then
+  ${CI_SOURCE_PATH}/${CI_PARENT_DIR}/ros_pre-release.sh
+  retval_prerelease=$?
+  if [ $retval_prerelease -eq 0 ]; then success; else error; fi  # Internally called travis_time_end for prerelease_from_travis_sh
+  # With Prerelease option, we want to stop here without running the rest of the code.
 fi
-travis_time_end  # prerelease_from_travis_sh
```

Comment:
Removing `travis_time_end` in this line is intended. I now let [`success` and `error` functions do the work internally](https://github.com/ros-industrial/industrial_ci/pull/50/files/bb704c7af752fa389e1e9e4a5ad990eab3b6812b#diff-a188fc26ef16ad3194c8d957e9e0cd3dR78).

Prior to this PR, `error` function already ran `travis_time_end`, but only when non-zero result occurred. `error` also run `exit 1`, which was called only with non-zero. So it was kind of redundant and incomplete. I hope this PR is cleaner and complete, in terms of fold and exit.